src/css/custom.css

```diff
@@ -503,6 +503,7 @@ html[data-theme="dark"] .header-dockerhub-link::before {
 /* Navbar formatting */
 .navbar {
   --ifm-navbar-height: 8vh;
+  height: clamp(60px, var(--ifm-navbar-height), 100px);
```

Comment:
Clamp has pretty solid cross-browser support these days (https://caniuse.com/css-math-functions), and browsers should just fall back to ```height: var(--ifm-navbar-height)```, which this rule is trying to overwrite.